core/src/services/mod.rs

```diff
@@ -312,3 +312,10 @@ mod seafile;
 pub use seafile::Seafile;
 #[cfg(feature = "services-seafile")]
 pub use seafile::SeafileConfig;
+
+#[cfg(feature = "services-upyun")]
+mod upyun;
+#[cfg(feature = "services-upyun")]
+pub use upyun::UpYun;
```

Comment:
Please call it `Upyun`, the naming should be exactly the same mapping.

- `abc` -> `abc` -> `Abc`
- `abc-def` -> `abc_def` -> `AbcDef`

Reply:
fixed